test/suites/device-mqtt/main_test.go

```diff
@@ -61,6 +58,10 @@ func TestMain(m *testing.M) {
 }
 
 func TestCommon(t *testing.T) {
+	utils.TestSecret(t, deviceMqttApp, utils.Secret{
+		TestManualConnection: needManualConnection,
+	})
+
```

Comment:
Does TestSecret run before all other tests? If not, the tests will fail when the snap is built locally.

Reply:
Yes, TestSecret runs before all other tests.